client/jetpack-connect/authorize.js

```diff
@@ -739,6 +740,7 @@ const connectComponent = connect(
 			user: getCurrentUser( state ),
 			userAlreadyConnected: getUserAlreadyConnected( state ),
 			partnerSlug: getPartnerSlugFromQuery( state ),
+			partnerID: getPartnerIdFromQuery( state ),
```

Comment:
If we follow the Calypso coding standards, we should rename this prop to `partnerId`. Just like it's done in the name of the existing selector.

Reply:
I would still like to have this addressed.